worker.go

```diff
@@ -43,6 +48,39 @@ func setSkipTLS(o *mqtt.ClientOptions) {
 	o.SetTLSConfig(oldTLSCfg)
 }
 
+func NewTLSConfig(ca, certificate, privkey []byte) (*tls.Config, error) {
+	// Import trusted certificates from CA
+	certpool := x509.NewCertPool()
+	ok := certpool.AppendCertsFromPEM(ca)
+
+	if !ok {
+		return nil, fmt.Errorf("CA is invalid")
+	}
+
+	// Import client certificate/key pair
+	cert, err := tls.X509KeyPair(certificate, privkey)
+	if err != nil {
+		return nil, err
+	}
+
+	// Create tls.Config with desired tls properties
+	return &tls.Config{
+		// RootCAs = certs used to verify server cert.
+		RootCAs: certpool,
+		// ClientAuth = whether to request cert from server.
+		// Since the server is set up for SSL, this happens
+		// anyways.
+		ClientAuth: tls.NoClientCert,
+		// ClientCAs = certs used to validate client cert.
+		ClientCAs: nil,
+		// InsecureSkipVerify = verify that cert contents
+		// match server. IP matches what is in cert etc.
+		InsecureSkipVerify: true,
```

Comment:
This should be set to `false`. Setting this to `true` has the effect that the stresser will accept any certificate presented by the server.